sdk/test-utils/perfstress/test/serviceClientGet.spec.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createPipelineRequest, PipelineRequest } from "@azure/core-rest-pipeline";
+import { ServiceClient } from "@azure/core-client";
+import { PerfStressTest, PerfStressOptionDictionary, drainStream } from "../src";
+import { getCachedHttpsAgent } from "../src/utils";
+
+interface ServiceClientGetOptions {
+  "first-run-extra-requests": number;
```

Comment:
normally we would use camel casing: `firstRunExtraRequests`.

Reply:
That was my first instinct, but it's blocked by this issue: https://github.com/Azure/azure-sdk-for-js/issues/17992.  The cross-language standard for the perf framework command-line options is `kebab-case`, and the only way to implement this currently in the JS perf framework is to use `kebab-case` for the property name itself.

Reply:
ohh

Reply:
Also prior art: https://github.com/Azure/azure-sdk-for-js/blob/7084ffbeb94c6d4970f8d341e3afcd21c7fcb9d1/sdk/test-utils/perfstress/src/options.ts#L67-69